src/server/modules/statistics/repositories/StatisticsRepository.ts

```diff
@@ -27,18 +27,21 @@ export class StatisticsRepository implements interfaces.StatisticsRepository {
     )
 
     if (userCount == null) {
-      userCount = await User.count()
+      // Allow use of read replica
+      userCount = await User.unscoped().count()
```

Comment:
What if we had an explicit `useReplica` scope specifying `useMaster: false` instead, and have the `defaultScope` be `useMaster: true`?